.github/workflows/ci.yml

```diff
@@ -0,0 +1,53 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - '*'
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: opendistroforelasticsearch/security-maven:v1
+
+    steps:
+
+    - name: Checkout security-parent
+      uses: actions/checkout@v1
+      with:
+        repository: opendistro-for-elasticsearch/security-parent
+        ref: refs/heads/master
+    - name: Install security-parent
+      run: mvn clean install -DskipTests --file ../security-parent/pom.xml
+        
+    - name: Checkout security
+      uses: actions/checkout@v1
+      with:
+        repository: opendistro-for-elasticsearch/security
+        ref: refs/heads/master
+    - name: Install security
+      run: mvn clean install -DskipTests --file ../security/pom.xml
+
+    - name: Checkout security-advanced-modules
+      uses: actions/checkout@v1
+
+    - name: Checkstyle
+      run: mvn checkstyle:checkstyle
```

Comment:
I thought we are skipping this?

Reply:
`mvn checkstyle:checkstyle` will report all the errors using default checkstyle but won't make the build job fail. Once we write our own checkstyle rules and use maven checkstyle plugin, we will run `mvn checkstyle:check` to make the job fail if it violates the standards.

Reply:
From github actions run: `[INFO] There are 5834 errors reported by Checkstyle 8.19 with sun_checks.xml ruleset.`

Reply:
Got it.